### PR TITLE
Update example install url from try.gogs.io to try.gitea.io

### PR DESCRIPTION
**Pull request by strk**

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Comment by bkcsoft** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -29,6 +29,9 @@
 
 The goal of this project is to make the easiest, fastest, and most painless way of setting up a self-hosted Git service. With Go, this can be done with an independent binary distribution across **ALL platforms** that Go supports, including Linux, Mac OS X, Windows and ARM.
 
+Want to try it before doing anything else?
+Do it [online](https://try.gitea.io/)!
```
Comment (tboerger) [inline README.md:33]: Again the new lines where it's not useful? ;)

**Comment by strk** [comment]
The new lines are useful to me. If they don't hurt you, why bother ?

File: README.md (hunk 1 above)
Comment (tboerger) [reply README.md:33]: IMHO it should be part of the previous paragraph.

**Comment by lunny** [comment]
Please send another PR.

**Pull request merged**